HTML5-frontend/ndt-library.js

```diff
@@ -0,0 +1,234 @@
+/**
+ * @fileoverview This javascript code includes an NDT object that contains the
```

Comment:
I think it's actually the reverse – the other file is the library and this is the interface that provides callbacks and m-lab-ns support.